### ISSUE TITLE
SSL error when trying to fetch data

### ISSUE THREAD

**Issue by DaemonLoki** [AvdLee/Roadmap#72]
## First things first

Thanks for the brilliant package, folks! The setup has been easy and I love the predefined styles!

## Setting

However, I'm running into an issue. I've defined the JSON locally and loaded it from the bundle. This works but when the screen appears it gives me SSL errors. The same happens when trying to tap the vote button.

## Error message

```
NSURLErrorDomain Code=-1200 "An SSL error has occurred and a secure connection
 to the server cannot be made." UserInfo{NSErrorFailingURLStringKey=
https://api.countapi.xyz/get/<my-bundle-id>/feature2, NSLocalizedRecoverySuggestion=
Would you like to connect to the server anyway?, _kCFStreamErrorDomainKey
=3,_NSURLErrorFailingURLSessionTaskErrorKey=LocalDataTask 
<0D7EC436-7228-497B-A1CF-28DCE78A28D3>.<3>, _NSURLErrorRelatedURLSessionTaskErrorKey=()
```

## Tried fixes

I thought it may be due to the fact that I need to allow to bypass App Transport Security (although it is an HTTPS domain) and both whitelisting everything as well as specifying the domain to be exempted doesn't work.

## Other thoughts

I'm not sure what other reasons it might have, maybe it is due to me having it only locally as files and it is therefore not correctly registered with the `countapi`?

Anyways: Looking forward to any other ideas you might have. Thanks again for the package!

**Comment by vdhamer** [comment]
Might be related to #70. It previously worked.

**Comment by jordibruin** [comment]
Yep, unfortunately countapi went down (again). Im in touch with their ceo, they're looking into it. Sorry about that!

Added the warning back.

**Comment by DaemonLoki** [comment]
No need to be sorry for anything. 
I'm just glad it wasn't me who messed things up. :) 

Thanks for the quick answers, and if there's anything you need help with let me know!

**Comment by jordibruin** [comment]
Unfortunately they won't be able to solve it anytime soon.

An update to the matter.
I checked with our team and we might not be able to restore this in the near future.
I realize this is not the response you were
hoping for and am sorry for any inconvenience caused.
Best,
Aleksa
Customer Success Manager

If anyone has any suggestions for alternatives that would be great!

**Comment by AvdLee** [comment]
Closing this as we no longer use CountAPI
